README.md

```diff
@@ -174,7 +174,7 @@ Parameters:
 | `project-id` | string | Polarion project ID  | `AppStudio`  |
 
 # Reporting issues
-For reporting issues with e2e tests please use [RHDP JIRA project](https://issues.redhat.com/projects/RHDP) - please use labels `appstudio` and `quality`. If the issues is also relevant to CI - for example it is blocking PRs, the also use label `appstudio-e2e-tests-known-issues`.
+For reporting issues with e2e tests please use [STONE Jira project](https://issues.redhat.com/browse/STONE) - please use labels `ci-fail` and `quality`. For existing issues, see [this Jira query](https://issues.redhat.com/issues/?jql=labels%20%3D%20%22ci-fail%22%20and%20labels%20%3D%20%22quality%22%20and%20resolution%20%3D%20unresolved).
```

Comment:
just a suggestion to extend the filter to include other jira projects where the jira could have been reported or was moved to
```suggestion
For reporting issues with e2e tests please use [STONE Jira project](https://issues.redhat.com/browse/STONE) - please use labels `ci-fail` and `quality`. For existing issues, see [this Jira query](https://issues.redhat.com/issues/?jql=project%20in%20(STONE%2C%20STONEBLD%2C%20HAC%2C%20HACBS%2C%20DEVHAS%2C%20RHDP%2C%20PLNSRVCE)%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20ci-fail).
```

Reply:
My Jira filter didn't specify projects at all. Of course there is a slight risk that some completely unrelated issue will have the same two labels, but right now it's not the case. Do you think listing the projects is better? The downside is if we learn later that there is some new project we need to add ;)

Reply:
oh, ok I missed that - I actually removed "quality" label from the filter (to list issues that failed on ci, but might be actually a product bug) and the list contained a lot of issues from totally unrelated jira projects. so that's why I specified them.

Reply:
ok let's leave it as is

Reply:
ok, thanks